cg/services/run_parameters_service/run_parameters_service.py

```diff
@@ -129,6 +129,10 @@ def _get_index_settings(self) -> IndexSettings:
             return NOVASEQ_6000_POST_1_5_KITS_INDEX_SETTINGS
         return NO_REVERSE_COMPLEMENTS_INDEX_SETTINGS
 
+    @abstractmethod
+    def get_flow_cell_model(self):
```

Comment:
As in the run parameters the term "model" is not used, maybe a docstring with some examples could be useful